base/docker-compose.yaml

```
@@ -209,6 +212,7 @@ services:
     image: ghcr.io/uwcirg/shl-ltt-server:${SHL_SERVER_IMAGE_TAG:-latest}
     environment:
       PUBLIC_URL: https://shl-server.${BASE_DOMAIN}
+      VERSION: ${SHL_SERVER_IMAGE_TAG:-"latest"}
```

This shouldn't need to be quoted, at least for YAML
```suggestion
      VERSION: ${SHL_SERVER_IMAGE_TAG:-latest}
```